run_tests.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import feedparser
+import sys
+import os
+
+feed = feedparser.parse('./feed.rss')
+
+absolute_address = "https://contravariance.rocks/"
+
+def validate_path(path):
+    replaced = path.replace(absolute_address, "")
+    # If file doesn't exist or is empty, return false
+    return os.path.getsize(replaced) != 0
+
+def fail(message):
+    print message
+    sys.exit(1)
+
+for entry in feed.entries:
+    print "Validating", entry.title
+    if not validate_path(entry["image"]["href"]):
+        fail("Could not find image %s" % (entry["image"]))
+    for enclosure in entry.enclosures:
+        if int(enclosure["length"] == 0):
+            fail("Invalid length for item '%s'" % (entry.title,))
+        if not validate_path(enclosure["href"]):
```

Comment:
Nit: store the `enclosure["href"]` in a variable to prevent having to repeat it in the next line?